subprojects/common/junit-utils/src/main/java/ru/avito/util/RandomUtils.kt

```diff
@@ -5,11 +5,13 @@ package ru.avito.util
 import java.util.ArrayList
 import java.util.Date
 import java.util.Random
+import java.util.concurrent.ThreadLocalRandom
 
 // TODO: to consider using reflection for automation.
 // For example - https://github.com/mtedone/podam
 
-private val random = Random(System.currentTimeMillis())
+private val random: ThreadLocalRandom
+    get() = ThreadLocalRandom.current()
 
```

Comment:
We refer to `Random` in kdoc in functions in this file. Need to fix it.

Reply:
Change